<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="34.198" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2023-02-08T03:32:10" tests="0" file="cypress\e2e\asset-list-tests\assetListSearchSpec.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Asset Management page search tests" timestamp="2023-02-08T03:32:10" tests="1" time="34.187" failures="1">
    <testcase name="Asset Management page search tests &quot;before each&quot; hook for &quot;verify search results for assets table&quot;" time="0.000" classname="&quot;before each&quot; hook for &quot;verify search results for assets table&quot;">
      <failure message="Timed out retrying after 30000ms: Expected &lt;div.spinner&gt; not to exist in the DOM, but it was continuously found.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Asset Management page searc...`" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 30000ms: Expected <div.spinner> not to exist in the DOM, but it was continuously found.

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Asset Management page searc...`
    at Context.eval (webpack:///./cypress/support/commands.js:228:3)]]></failure>
    </testcase>
  </testsuite>
</testsuites>